stoq/core.py

```diff
@@ -236,6 +237,8 @@ def load_config(self):
                     value = tuple(i.strip() for i in value.split(","))
                 elif opt.endswith("_int"):
                     value = int(value.strip())
+                elif opt.endswith("_bool"):
```

Comment:
This is actually done already..no need to set it explicitly. First stoq will try to load it as a `bool`, otherwise it will go through the others. This goes for in `plugins.py` also.